src/components/auth/LoginPage.scss

```diff
@@ -0,0 +1,24 @@
+.LoginForm {
+    margin-top: 20px;
+    width: 80%;
+    max-width: 540px;
+    padding: 0 5%;
+
+    h1 {
+        font-weight: bold;
+    }
+
+    a {
+        color: lightslategray;
+    }
+
+    button {
+        background-color: green;
+        margin-top: 1.5em;
+        font-weight: bold;
+        margin-left: auto;
+        margin-right: auto;
+        padding: 1rem 2rem;
+        border-radius: .3rem;
+    }
+}
```

Comment:
Don't forget to add line breaks after each file.
VSCode should add these automatically if you configure it to do so.
ESLint would complain if this happened in a JS file.